Apps/Patient/test/unit/Services/PatientDataServiceTests.cs

```diff
@@ -21,137 +21,216 @@ namespace HealthGateway.PatientTests.Services
     using System.Text.Json;
     using System.Threading;
     using System.Threading.Tasks;
+    using AutoMapper;
+    using HealthGateway.Common.Models.PHSA;
     using HealthGateway.Common.Services;
     using HealthGateway.Patient.Constants;
+    using HealthGateway.Patient.Models;
     using HealthGateway.Patient.Services;
     using HealthGateway.PatientDataAccess;
+    using HealthGateway.PatientTests.Utils;
     using Moq;
     using Shouldly;
     using Xunit;
+    using DiagnosticImagingExam = HealthGateway.Patient.Services.DiagnosticImagingExam;
+    using DiagnosticImagingStatus = HealthGateway.PatientDataAccess.DiagnosticImagingStatus;
+    using OrganDonorRegistration = HealthGateway.Patient.Services.OrganDonorRegistration;
+    using OrganDonorRegistrationStatus = HealthGateway.PatientDataAccess.OrganDonorRegistrationStatus;
     using PatientDataQuery = HealthGateway.Patient.Services.PatientDataQuery;
-    using PatientFileQuery = HealthGateway.Patient.Services.PatientFileQuery;
+    using PatientFileQuery = HealthGateway.PatientDataAccess.PatientFileQuery;
```

Comment:
This one is annoying, I've manually changed them but the clean up process keeps changing it, so I've forced it to accept by reversing the auto cleanup, but no guarantee this will stay like this.

I made some attempts to do specifically what you've asked here but I'm not sure if what you've reported was auto-f*ed or not.